arch/risc-v/src/mpfs/Kconfig

```diff
@@ -388,6 +388,43 @@ config MPFS_IHC_RPMSG_CH2
 		Use this only if using 2 x RPMSG channels.  This makes the NuttX work as the
 		RPMSG channel 2 client.
 
+config MPFS_IHC_WITH_HSS
+	bool "IHC with HSS"
+	depends on MPFS_IHC_CLIENT
+	default n
+	---help---
+		Set this to true if using IHC with HSS bootloader
+
+config MPFS_CH1_VRING_SHMEM_ADDR
+	hex "Channel 1 Vring shared memory start"
+	default 0xA2410000
```

Comment:
Could we  place these to L2LIM? i.e. reduce the bootloader's ram size (currently l2lim  (rwx)  : ORIGIN = 0x08000000, LENGTH = 1024k) and use the rest of the l2lim for these? would it work?

Reply:
I will create Jira task about this.